From: Tom Tromey <tromey at redhat dot com>
Subject: Patch: RFA: fix PR java/21519

There is a small bug in save_expr handling when writing bytecode.
emit_load calls NOTE_PUSH for us, so we don't need to call it
ourselves.  This only gets triggered by "gcj -C -O2".

	PR java/21519:
	* jcf-write.c (generate_bytecode_insns) <SAVE_EXPR>: Don't call
	NOTE_PUSH.

--- gcc/gcc/java/jcf-write.c 14 Feb 2005 20:04:36 -0000
+++ gcc/gcc/java/jcf-write.c 13 May 2005 21:19:21 -0000
@@ -2235,7 +2235,6 @@
 	{
 	  tree type = TREE_TYPE (exp);
 	  emit_load (arg, state);
-	  NOTE_PUSH (TYPE_IS_WIDE (type) ? 2 : 1);
 	}
       break;
     case CONVERT_EXPR:
